types/yargs/v11/.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*.ts]
```

Comment:
Please don't check this in

Reply:
It's only there because the original typings for v11, v10 and v8 had it. Removed in 730926d5fe99c1f190aaddd9b7cbdad894ceea14